refresher.go

```diff
@@ -51,7 +55,16 @@ func (r *Refresher) launchIfNecessary(name string, value interface{}, f func(v i
 }
 
 func (r *Refresher) ReadTag(name string, value interface{}) error {
-	r.launchIfNecessary(name, value, func(v interface{}) { r.plc.ReadTag(name, v) })
+	r.launchIfNecessary(name, value, func(v interface{}) {
+		err := r.plc.ReadTag(name, v)
+		if err != nil {
+			if r.ErrorCallback != nil {
+				r.ErrorCallback(err)
+			} else {
+				fmt.Println("WARNING: Unhandled plc.Refresher.ReadTag error:", err)
```

Comment:
Suggestion: migrate to using `log`

Reply:
Sure, that makes more sense. The `Println` is just a last resort reminder that errors weren't handled properly, but `log` is better.

Reply:
@dijkstracula Matthias suggested that I should delete the `Println` entirely. A user should be free to ignore errors without getting spammed by debug.

Reply:
...so I've pushed another fixup with that change.

Reply:
That's reasonable. 👍